modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -120,17 +123,45 @@ storage:
           type=ethernet
           interface-name=ens2
           [ipv4]
-          address1=10.10.10.10/24,10.10.10.1
+          address1=10.10.10.10/24
           dns=8.8.8.8;
           dns-search=
           may-fail=false
           method=manual
 ----
 
+NMstate
+[source, yaml]
+----
+variant: fcos
+version: 1.4.0
+storage:
+  files:
+    - path: /etc/nmstate/ens2.yml
+      mode: 0600
+      contents:
+        inline: |
+          hostname:
+            config: myhostname
+          dns-resolver:
+            config:
+              search: []
+              server:
+              - 8.8.8.8
+          interfaces:
+          - name: ens2
+            type: ethernet
+            wait-ip: ipv4
+            ipv4:
+              enable: true
+              address:
+              - ip: 10.10.10.10
+                prefix-length: 24
```

Comment:
the later examples include a `routes:` entry in the nmstate yml file. I'm not sure I understand why those include it and this one doesn't?